### ISSUE TITLE
The magnitude of multiple training losses

### ISSUE THREAD

**Issue by RenShuhuai-Andy** [obvious-research/phenaki-cvivit#4]
Hi, thanks for the great work.

Can you kindly provide the magnitude of multi losses (e.g., perceptual loss, i3d loss) during the training or after the convergence? It would be better if you can provide the figure of loss curves. This information can help us validate the correctness of our training procedure. Thanks a lot.

**Comment by obvious-research** [comment]
Hello, 

commit_loss_w=[0.1]  # commit loss weight
gen_loss_w=[0.]  # generator loss weight
perceptual_loss_w=[1.]  # vgg loss weight
i3d_loss_w=[1.]  # i3d loss weight
recon_loss_w=[10.]  # reconstruction loss weight
use_discr=[0] # use stylegan loss or not

Here are the parameters we used !

This might be useful as well : 

learning_rates = [1e-4]  # Learning rate
weight_decays = [1e-4]  # Weight decay

Hope that helps !
